tests/observability/metrics/utils.py

```diff
@@ -1342,3 +1352,87 @@ def get_vmi_guest_os_kernel_release_info_metric_from_vm(
         NODE_STR: vm.vmi.virt_launcher_pod.node.name,
         "vmi_pod": vm.vmi.virt_launcher_pod.name,
     }
+
+
+def get_pvc_size_bytes(vm: VirtualMachineForTests) -> str:
+    assert vm.instance.spec.dataVolumeTemplates or len(vm.instance.spec.dataVolumeTemplates) > 0, (
+        "VM has no DataVolume templates"
+    )
+    return str(
+        int(
+            bitmath.parse_string_unsafe(
+                PersistentVolumeClaim(
+                    name=vm.instance.spec.dataVolumeTemplates[0].metadata.name,
+                    namespace=vm.namespace,
+                ).instance.spec.resources.requests.storage
+            ).Byte.bytes
+        )
+    )
+
+
+def get_vm_virt_launcher_pod_requested_memory(vm: VirtualMachineForTests) -> int:
+    containers = vm.vmi.virt_launcher_pod.instance.spec.containers
+    assert containers, f"No containers found in virt-launcher pod of {vm.vmi.virt_launcher_pod.name}"
+    return int(bitmath.parse_string_unsafe(containers[0].resources.requests.memory).bytes)
+
+
+def virt_launcher_pod_metrics_resource_exists(vm_for_test: VirtualMachineForTests) -> None:
```

Comment:
```suggestion
def wait_for_virt_launcher_pod_metrics_resource_exists(vm_for_test: VirtualMachineForTests) -> None:
```
function name should indicate that there's a sampler waiting for a condition